api/lib/block-listeners/base-node-subscription.js

```diff
@@ -135,67 +133,29 @@ class BaseNodeSubscription {
   }
 
   async getValidators(block, dataSource) {
-    dataSource.getAllValidators(block.height).then((validators) => {
-      this.store.update({
+    dataSource.getValidators(block.height).then(async (validators) => {
+      await this.store.update({
         validators: validators
       })
       const storeValidators = Object.values(this.store.validators)
```

Comment:
is this different then `validators` in line 136?

Reply:
no, I can remove the duplicate